MAUI/Cartesian-Charts/PlotBand.md

```diff
@@ -0,0 +1,334 @@
+                </chart:NumericalAxis.PlotBands>
+            </chart:NumericalAxis>
+        </chart:SfCartesianChart.YAxes>
+        . . .
+    </chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfCartesianChart chart = new SfCartesianChart();
+    . . .
+    
+    NumericalAxis numericalAxis = new NumericalAxis();
+    NumericalPlotBandCollection numericalPlotBandCollection = new NumericalPlotBandCollection();
+    NumericalPlotBand plotBand = new NumericalPlotBand
+    {
+        Start = 28,
+        Size = 10,
+        Fill = Colors.Orange
+    };
+
+    numericalPlotBandCollection.Add(plotBand);
+    numericalAxis.PlotBands = numericalPlotBandCollection;
+    chart.YAxes.Add(numericalAxis);
+    ...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Plot band with numerical axis in MAUI Cartesian chart](Plotband_images/Numerical.png)
+
+## Add plot band for dateTime axis
+
+Plot band can be added to the date-time axis by specifying date values to the [Start]() and [Size]() properties.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:SfCartesianChart>
+        . . .
+     <chart:SfCartesianChart.XAxes>
+         <chart:DateTimeAxis>
+          <chart:DateTimeAxis.PlotBands>
+             <chart:DateTimePlotBandCollection >
+                 <chart:DateTimePlotBand Start="2024-01-24"
+                                         Size="2"
+                                         SizeType="Days"
+                                         Fill="Orange"/>
+                    </chart:DateTimePlotBandCollection >
+                </chart:DateTimeAxis.PlotBands>
+            </chart:DateTimeAxis>
+        </chart:SfCartesianChart.XAxes>
+
+        <chart:SfCartesianChart.YAxes>
+            <chart:NumericalAxis/>
+        </chart:SfCartesianChart.YAxes>
+    </chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfCartesianChart chart = new SfCartesianChart();
+    . . .
+    DateTimeAxis dateTimeAxis = new DateTimeAxis();
+    DateTimePlotBandCollection dateTimePlotBandCollection = new DateTimePlotBandCollection();
+    DateTimePlotBand plotBand = new DateTimePlotBand
+    {
+        Start = new DateTime(2024, 1, 24),
+        Size = 2,
+        WidthType = DateTimeUnits.Days,
+        Fill = Color.Orange
+    };
+
+    dateTimePlotBandCollection.Add(plotBand);
+    dateTimeAxis.PlotBands = dateTimePlotBandCollection;
+    chart.XAxes.Add(dateTimeAxis);
+
+    NumericalAxis secondaryAxis = new NumericalAxis();
+    chart.YAxes.Add(secondaryAxis);
+    ...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Plot band with numerical axis in MAUI Cartesian chart](Plotband_images/)
+
+## Recursive plot band
+
+This feature is used to enable the plot bands to be drawn repeatedly at the regular intervals. This will be useful when you need to mark an event that occurs recursively along the timeline of the chart. The following properties are used to configure this feature:
```

Comment:
This feature is used to enable the plot bands to be drawn repeatedly at regular intervals. This will be useful when you need to mark an event that occurs recursively along the timeline of the chart. The following properties are used to configure this feature: